reloj-checador/app/models/attendace.rb

```diff
@@ -1,3 +1,17 @@
 class Attendace < ApplicationRecord
-    validates :private_number, presence: true
+  validates :private_number, presence: true
+
+  after_create :set_check_type
```

Comment:
I'd say `before_validation` would be a better place, so you avoid saving the record twice

Reply:
Comment taken and changed behavior; Running migration and deleted both fields (check_in and check_out; created instead a new field check_type wich validate and set when a user is entering (IN) o leaving (OUT)